cve_bin_tool/version_scanner.py

```diff
@@ -168,6 +170,11 @@ def scan_file(self, filename):
             java_lines = "\n".join(lines.splitlines())
             yield from self.run_java_checker(filename, java_lines)
 
+        # Javascript checker
+        if output and "package-lock.json" in output:
+            javascript_lines = "\n".join(lines.splitlines())
```

Comment:
Is this line necessary? It removes newline from `lines` and adds newline back again right? Or am I missing something?
The same is true in line 170.

Reply:
@BreadGenie I have been following the same pattern (and code) for all of the language scanners but you are right, the lines parameter isn't used for the javascript scanner and can probably be removed.